doc/examples.xml

```diff
@@ -743,3 +1393,193 @@ true
+    graph are the numbers from 1 to <M>n</M> in binary representation, with a 
+    vertex <M>v</M> having as a direct parent the vertex with binary
+    representation the same as <M>v</M> but with the lowest 1-bit cleared. For 
+    example, the vertex <M>011</M> has parent <M>010</M>, and the vertex
+    <M>010</M> has parent <M>000</M>.<P/>
+
+    If the optional first argument <A>filt</A> is present, then this should
+    specify the category or representation the digraph being created will
+    belong to. For example, if <A>filt</A> is <Ref Filt="IsMutableDigraph"/>,
+    then the digraph being created will be mutable, if <A>filt</A> is <Ref
+    Filt="IsImmutableDigraph"/>, then the digraph will be immutable. 
+    If the optional first argument <A>filt</A> is not present, then <Ref
+    Filt="IsImmutableDigraph"/> is used by default.<P/>
+
+    See <URL>https://metacpan.org/pod/Graph::Maker::BinomialTree</URL> for
+    further details. 
+
+<Example><![CDATA[
+gap> D := BinomialTreeGraph(9);
+<immutable acyclic digraph with 9 vertices, 16 edges>]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="CycleGraph">
+<ManSection>
+  <Oper Name="CycleGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is an integer greater than 2 then this operation returns the
+    <M>n</M>th <E>Cycle graph</E>, consisting of the cycle on <M>n</M> vertices.
+    The Cycle graph, unlike the Cycle digraph, is symmetric. The Cycle graph 
+    has <M>n</M> vertices and <M>n</M> undirected edges. The Cycle graph is 
+    simple so the non-simple graphs with a single vertex and single loop and
+    with two vertices and two edges between them are excluded.<P/>
+
+    If the optional first argument <A>filt</A> is present, then this should
+    specify the category or representation the digraph being created will
+    belong to. For example, if <A>filt</A> is <Ref Filt="IsMutableDigraph"/>,
+    then the digraph being created will be mutable, if <A>filt</A> is <Ref
+    Filt="IsImmutableDigraph"/>, then the digraph will be immutable. 
+    If the optional first argument <A>filt</A> is not present, then <Ref
+    Filt="IsImmutableDigraph"/> is used by default.<P/>
+
+    See <URL>https://mathworld.wolfram.com/CycleGraph.html</URL> for
+    further details. 
+
+<Example><![CDATA[
+gap> D := CycleGraph(7);
+<immutable symmetric digraph with 7 vertices, 14 edges>]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="PathGraph">
+<ManSection>
+  <Oper Name="PathGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is a non-negative integer then this operation returns the
+    <M>n</M>th <E>Path graph</E>, consisting of the path on <M>n</M> vertices.
+    This is similar to the Chain digraph but with symmetric edges. The Path 
+    graph has <M>n</M> vertices and <M>n-1</M> edges. The graph is acyclic.<P/>
```

Comment:
Incorrect, non-empty symmetric digraphs are not acyclic. In particular, for any edge x→y, x→y→x is a cycle.